v3/setup/capcom.sh

```diff
@@ -7,26 +7,24 @@ if [ "${NODE_ROLE}" != "control" ]  && [ "${NODE_ROLE}" != "it-hybrid" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get capcom-bootstrapped)" == "true" ]; then
+if [ "$(etcdctl get /capcom/config/bootstrapped)" == "true" ]; then
```

Comment:
What about a `/state` path for each service? Like `/capcom/state/bootstrapped` or `/flight-director/state/xyz`, etc.